### ISSUE TITLE
Support for Mobile Web

### ISSUE THREAD

**Issue by DaKaZ** [GeraudBourdin/Ti.SvgView#10]
In mobile web, you can simply create a standard image view.  So my code is littered with blocks like these:

```
if (!OS_MOBILEWEB) {
	var svgView = require('com.geraudbourdin.svgview');
	var svg = svgView.createView({
	    image: "/images/logo.svg",
	    width: Ti.UI.FILL,
	    left: "15%",
	    right: "15%",
	    top: "33%"
	});
	$.viewLogo.add(svg);
} else {
	$.viewLogo.add(Ti.UI.createImageView({
		image: "/images/logo.svg",
	    width: Ti.UI.FILL,
	    left: "15%",
	    right: "15%",
	    top: "33%"
	}));
}
```

It would be ideal if this module would do that automatically.  I started to look at how to do this, but I have never built a module so frankly I am at a loss.  Happy help if someone could point me to a primer or howto on this subject.

**Comment by GeraudBourdin** [comment]
you are right, that would be great to get this done. I do not have enough time at the moment (too much work, sadly), but i ll surely digg it in the future. That would have been great if somone could do this quicly. I'll look at the way to do it in a kind of module.. Thanks for the issue :)

**Comment by DaKaZ** [comment]
Hi @GeraudBourdin - I happy to do the work, I just really don't know where to start with a module.  If you have any suggestions at all I can look into.
